src/main/kotlin/rys/ajaxpetproject/configuration/MongoConfig.kt

```diff
@@ -0,0 +1,70 @@
+package rys.ajaxpetproject.configuration
+
+import org.springframework.boot.CommandLineRunner
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.ComponentScan
+import org.springframework.context.annotation.Configuration
+import org.springframework.data.mongodb.repository.config.EnableMongoRepositories
+import rys.ajaxpetproject.model.MongoChat
+import rys.ajaxpetproject.model.MongoMessage
+import rys.ajaxpetproject.model.MongoUser
+import rys.ajaxpetproject.repository.ChatRepository
+import rys.ajaxpetproject.service.ChatService
+import rys.ajaxpetproject.service.MessageService
+import rys.ajaxpetproject.service.UserService
+import java.util.*
+
+@Configuration
+@EnableMongoRepositories(basePackageClasses = [ChatRepository::class])
+@ComponentScan(basePackageClasses = [ChatRepository::class])
+class MongoConfig {
+
+    @Bean
+    fun init( userService: UserService,  chatService: ChatService,  messageService: MessageService): CommandLineRunner {
```

Comment:
usually, such stuff is performed by specific tools. it can be left like is for test purposes but fyi checkout https://mongock.io/